individual/apps.py

```diff
@@ -97,6 +99,25 @@ def ready(self):
         self.__initialize_custom_filters()
         self._set_up_workflows()
         self.__register_masking_class()
+        self.__connect_signals()
+
+    def __connect_signals(self):
+        from core.models import ModuleConfiguration
+        post_save.connect(
+            self._reload_module_config,
+            sender=ModuleConfiguration,
+            weak=False
+        )
+
+    def _reload_module_config(self, sender, instance, **kwargs):
+        if instance.module == self.name and instance.layer == 'be':
+            # Reload the schema from the new config
+            IndividualConfig.individual_schema = json.loads(instance.config).get('individual_schema', '{}')
```

Comment:
Why not reloading all config ?

Reply:
because that would also require similar changes in MaskingClassRegistryPoint and possibly workflows to facilitate reloading. I don't currently have concrete use cases for reloading other configs. We can add them in the similar manner when the need arises. I've added TODOs to flag them.

Reply:
Done for the simple configs, but masking config dynamic reload would also require similar changes in MaskingClassRegistryPoint. I don't currently have concrete use cases for reloading the masking configs. We can add it in the similar manner when the need arises. I've added a TODO to flag it.